tests/backup/backup_license_test.go

```diff
@@ -0,0 +1,201 @@
+			err = Inst().S.RemoveLabelOnNode(destinationClusterWorkerNodes[0], "portworx.io/nobackup")
+			if err == nil {
+				destinationClusterWorkerNodeLabelRemoved = true
+			}
+			log.FailOnError(err, fmt.Sprintf("Failed to remove label portworx.io/nobackup=true from worker node %v", destinationClusterWorkerNodes[0].Name))
+			log.InfoD("Switching cluster context back to source cluster")
+			err = SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+			licenseCountUpdateCheck = func() (interface{}, bool, error) {
+				licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+				if err != nil {
+					return "", false, err
+				}
+				licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+				if licenseResponseInfoFeatureInfo[0].Consumed == int64(len(totalNumberOfWorkerNodes)) {
+					return "", false, nil
+				}
+				return "", true, nil
+			}
+			_, err = task.DoRetryWithTimeout(licenseCountUpdateCheck, licenseCountUpdateTimeout, licenseCountUpdateRetryTime)
+			licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+			log.InfoD("License count after removing label portworx.io/nobackup=true from node %v is %v ", destinationClusterWorkerNodes[0].Name, licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()[0].Consumed)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after removing label portworx.io/nobackup=true from node %v", destinationClusterWorkerNodes[0].Name))
+		})
+	})
+	JustAfterEach(func() {
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		if destinationClusterWorkerNodeLabelRemoved == false && destinationClusterWorkerNodeLabelApplied == true {
+			SetDestinationKubeConfig()
+			log.InfoD("Removing the applied label portworx.io/nobackup=true from worker nodes on destination cluster at the end of the testcase")
+			err = Inst().S.RemoveLabelOnNode(destinationClusterWorkerNodes[0], "portworx.io/nobackup")
+			dash.VerifySafely(err, nil, fmt.Sprintf("Failed to remove label portworx.io/nobackup=true from worker node %v", destinationClusterWorkerNodes[0].Name))
+		}
+		log.InfoD("Switching cluster context back to source cluster")
+		err = SetSourceKubeConfig()
+		log.FailOnError(err, "Switching context to source cluster")
+		if sourceClusterWorkerNodeLabelRemoved == false && sourceClusterWorkerNodeLabelApplied == true {
+			log.InfoD("Removing the applied label portworx.io/nobackup=true from worker nodes on source cluster at the end of the testcase")
+			err = Inst().S.RemoveLabelOnNode(sourceClusterWorkerNodes[0], "portworx.io/nobackup")
+			dash.VerifySafely(err, nil, fmt.Sprintf("Failed to remove label portworx.io/nobackup=true from worker node %v", sourceClusterWorkerNodes[0].Name))
+		}
```

Comment:
Same here, write a common method for this as well.

Reply:
same